<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SumedhKrishna" failures="3" tests="9" name="EDGE" time="71.474" errors="0" timestamp="2023-09-06T12:02:22 EDT">
  <testcase classname="com.tekarck.testCases.TC01_LoginErrorMessage" name="validateLoginToTheApplnWithBlankPassword" time="0.763"/>
  <testcase classname="com.tekarck.testCases.TC02_ValidLoginToSalesForce" name="loginToSalesForceAppUsingDataProviders" time="5.225"/>
  <testcase classname="com.tekarck.testCases.TC02_ValidLoginToSalesForce" name="loginToSalesForceAppUsingDataProviders" time="6.602">
    <failure type="java.lang.AssertionError" message="Failed to login for User Name and password : : anusha@tek.com  Sumedh expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed to login for User Name and password : : anusha@tek.com  Sumedh expected [true] but found [false]
at com.tekarck.testCases.TC02_ValidLoginToSalesForce.loginToSalesForceAppUsingDataProviders(TC02_ValidLoginToSalesForce.java:178)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginToSalesForceAppUsingDataProviders -->
  <testcase classname="com.tekarck.testCases.TC02_ValidLoginToSalesForce" name="loginToSalesForceAppUsingDataProviders" time="5.575">
    <failure type="java.lang.AssertionError" message="Failed to login for User Name and password : : anusha@tek  Sumedh@03 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed to login for User Name and password : : anusha@tek  Sumedh@03 expected [true] but found [false]
at com.tekarck.testCases.TC02_ValidLoginToSalesForce.loginToSalesForceAppUsingDataProviders(TC02_ValidLoginToSalesForce.java:178)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginToSalesForceAppUsingDataProviders -->
  <testcase classname="com.tekarck.testCases.TC02_ValidLoginToSalesForce" name="loginToSalesForceAppUsingDataProviders" time="5.722">
    <failure type="java.lang.AssertionError" message="Failed to login for User Name and password : : anusha@t.com  Anusha expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed to login for User Name and password : : anusha@t.com  Anusha expected [true] but found [false]
at com.tekarck.testCases.TC02_ValidLoginToSalesForce.loginToSalesForceAppUsingDataProviders(TC02_ValidLoginToSalesForce.java:178)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginToSalesForceAppUsingDataProviders -->
  <testcase classname="com.tekarck.testCases.TC03_CheckRememberMe" name="loginAndCheckRememberMe" time="8.485"/>
  <testcase classname="com.tekarck.testCases.TC04A_TestForgotPassword" name="forgotPwdLinkValidation" time="0.615"/>
  <testcase classname="com.tekarck.testCases.TC04B_InvalidUserNameAndPassword" name="loginToAppWithInvalidCreds" time="11.563"/>
  <testcase classname="com.tekarck.testCases.TC05_ValidateUserMenuDropDown" name="validateUserMenuDropDown" time="4.822"/>
</testsuite> <!-- EDGE -->
